pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -604,6 +604,12 @@ func (m *kubeGenericRuntimeManager) SyncPod(pod *v1.Pod, _ v1.PodStatus, podStat
 		podIP = podStatus.IP
 	}
 
+	rawHostIP, err := m.runtimeHelper.GetHostIP()
+	hostIP := rawHostIP.String()
```

Comment:
lol.. turns out there is a field on the Kubelet already for host IP https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet.go#L465. Seems like its mainly used for node status updates, not sure if the IP of the node will always match pod.Status.HostIP.